src/NServiceBus.NHibernate/NServiceBus.NHibernate.csproj

```diff
@@ -15,7 +15,8 @@
     <SignAssembly>true</SignAssembly>
     <TargetFrameworkProfile />
     <SolutionDir Condition="$(SolutionDir) == '' Or $(SolutionDir) == '*Undefined*'">..\</SolutionDir>
-    <NuGetPackageImportStamp>ec59a453</NuGetPackageImportStamp>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
```

Comment:
I did the package updates via vs15, not sure why that stuff is left in there. But do we really wanna fiddle around with each csproj which updates some packages?